config/application.rb

```diff
@@ -10,7 +10,9 @@ module TestGuru
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
+    #Для поиска версии через Rails.configuration.version
     config.version = File.read("#{Rails.root}/config/initializers/version.rb")
+    config.time_zone = 'Europe/Moscow'
```

Comment:
```suggestion
    config.time_zone = 'Moscow'
```
достаточно такого варианта